app/screens/ChildSetup.tsx

```diff
@@ -183,16 +187,30 @@ const ChildSetup = ({ navigation }: Props): any => {
   };
   const importFromFile = async (): Promise<any> => {
     if (Platform.OS == "android") {
```

Comment:
Can you refactor code, something like:

// Call the appropriate import function based on the platform
if (Platform.OS === 'android') {
  importDataAndroid();
} else {
  importDataNonAndroid();
}

Reply:
okay @Akhror sure, I will refactor above code